===================================================================
RCS file: /cvsroot/gaim/gaim/src/account.c,v
retrieving revision 1.91
retrieving revision 1.92
diff -u -r1.91 -r1.92
--- gaim/gaim/src/account.c	2004/06/15 02:37:22	1.91
+++ gaim/gaim/src/account.c	2004/06/29 12:29:22	1.92
@@ -1253,7 +1253,7 @@
 		fprintf(fp, "   <setting name='%s' type='int'>%d</setting>\n",
 				name, setting->value.integer);
 	}
-	else if (setting->type == GAIM_PREF_STRING) {
+	else if (setting->type == GAIM_PREF_STRING && setting->value.string != NULL) {
 		fprintf(fp, "   <setting name='%s' type='string'>%s</setting>\n",
 				name, setting->value.string);
 	}
